com_migratetojoomla/admin/tmpl/main/default.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_migratetojoomla
+ *
+ * @copyright     (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license       GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Router\Route;
+// No direct access to this file
+defined('_JEXEC') or die('Restricted Access');
+/** @var \Joomla\Component\Banners\Administrator\View\Banner\HtmlView $this */
+
+/** @var Joomla\CMS\WebAsset\WebAssetManager $wa */
+$wa = $this->document->getWebAssetManager();
+$wa->useScript('com_migratetojoomla.admin-migratetojoomla')
+    ->useScript('keepalive');
+
+?>
+<style>
```

Comment:
Don't use inline styles. It's ok for testing but you should have a media folder for the extension where you put all styles in, it's better house keeping. Use the web asset manager to load the files.